scanpy/neighbors/__init__.py

```diff
@@ -108,11 +107,15 @@ def neighbors(
     adata.uns['neighbors']['connectivities'] = neighbors.connectivities
     if neighbors.rp_forest is not None:
         adata.uns['neighbors']['rp_forest'] = neighbors.rp_forest
-    logg.info('    finished', time=True, end=' ' if _settings_verbosity_greater_or_equal_than(3) else '\n')
-    logg.hint(
-        'added to `.uns[\'neighbors\']`\n'
-        '    \'distances\', distances for each pair of neighbors\n'
-        '    \'connectivities\', weighted adjacency matrix')
+    logg.info(
+        '    finished',
+        time=True,
```

Comment:
OK, you're assuming that any previous `logg.info` call starts the timer. But that means that everything in between has to be logged with `logg.debug`. But often, we also want to logg out info also within the function. Hm. What do you think?

Reply:
I switched to explicit time passing

Reply:
Great!